src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotResolvedJavaMethod.java

```diff
@@ -58,6 +58,13 @@ public interface HotSpotResolvedJavaMethod extends ResolvedJavaMethod {
      */
     boolean hasReservedStackAccess();
 
+    /**
+     * Returns true if this method has a {@code Scoped} annotation.
```

Comment:
Can you please make this a qualified name: `jdk.internal.misc.ScopedMemoryAccess.Scoped`.
That makes it easier for someone not familiar with the code base to find.

Reply:
Done